_data/toc/page-builder.yml

```diff
@@ -141,6 +141,34 @@ pages:
     url: /page-builder/docs/how-to/how-to-deactivate-pagebuilder.html
     versionless: true
 
+- label: BlueFoot content migration
+  versionless: true
+  children:
+
+  - label: Overview of content migration
+    url: /page-builder-migration/docs/
+    versionless: true
+
+  - label: Install the migration module
+    url: /page-builder-migration/docs/install-migration-module.html
```

Comment:
Should all of these links also be extensionless like line 149?

Reply:
No, just the index.html, unless they can be extensionless?